packages/amplify-cli/src/init-steps/s0-analyzeProject.ts

```diff
@@ -191,6 +191,10 @@ async function getEditor(context: $TSContext) {
     editor = await editorSelection(editor);
   }
 
+  if (!editor) {
+    editor = context.exeInfo.localEnvInfo.defaultEditor;
```

Comment:
Last thing - but `exeInfo` has a type `any`, so I think `localEnvInfo` would benefit from optional chaining in the event that it's undefined.